streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -125,7 +116,7 @@ public StateStoreSupplier build() {
          * @return the interface used to specify the type of values; never null
          */
         public ValueFactory<Integer> withIntegerKeys() {
-            return withKeys(new IntegerSerializer(), new IntegerDeserializer());
+            return withKeys(Serdes.Integer());
```

Comment:
Do we also want to introduce `withDoubleKeys` and `withDoubleValues`, now that we have a serde for `Double`?